conda_build_config.yaml

```diff
@@ -1,6 +1,15 @@
+boost:
+  - 1.63.0
+  - 1.65.0
+  - 1.69.0
+  - 1.70.0
+  - 1.71.0
+  - 1.72.0
+  - 1.73.0
+  - 1.74.0
```

Comment:
Wow! Do you really need to support all those? I'm fine with it but this can be a problem further down the road as we don't update the older boost versions.

Reply:
Changes to any file outside of `recipe` should be removed

Reply:
well working through a pile of old packages and trying to cut it of all these static dependencies and link up with shared libs. So far built and tested well with all of them. I have hope that once that legacy is organised it will be possible to drop the old ones..

Reply:
Ah! Thanks @isuruf!

@mathause can you send a PR to remove that file? I thought it was in the recipe directory. Sorry.

Reply:
Actually. @isuruf just did it in https://github.com/conda-forge/staged-recipes/pull/12985. Thanks @isuruf!